src/ResXManager.View/Visuals/TranslationsViewModel.cs

```diff
@@ -122,11 +122,27 @@ private void Apply(IEnumerable<ITranslationItem> items)
         {
             var prefix = Configuration.EffectiveTranslationPrefix;
 
+            var prefixField = Configuration.PrefixFieldType;
+
             foreach (var item in items.Where(item => !string.IsNullOrEmpty(item.Translation)).ToArray())
             {
-                if (!item.Apply(prefix))
-                    break;
-
+                if (prefixField.HasFlag(PrefixFieldType.Both))
+                {
+                    if (!item.UpdateValue(prefix) || !item.UpdateComment(prefix))
+                        break;
+                }
+                else if (prefixField.HasFlag(PrefixFieldType.Comment))
+                {
+                    // We need to update the value here, otherwise it won't get the translation

```

Comment:
Maybe better extend the Apply methods arguments with the new field type parameter, and move this code inside the apply method, so you don't need to make such workarounds that need a comment.

Reply:
I agree that would be more ideal. The struggle I had with that approach was I couldn't figure out which project I should put the `PrefixFieldType`  enum in. I was running into some circular dependencies issues, but I can look into it a little more, unless you have any suggestions.

Reply:
I moved the Enum to `ResXManager.Infrastructure`, but this meant I wasn't able to have the `LocalizedDisplayName` attribute anymore. What do you think?

Reply:
I see; 
we could just pass two strings "textPrefix" and "commentPrefix" to the apply method, where they are just empty if not configured.

Reply:
I like the idea of the 2 prefix values. It provides the same functionality while preserving the same parameter types. Updated 👍